### DISCUSSION TITLE
version 9.4.1 `npm run build` is not working on digitalocean (ubuntu). Says: Error: > Build directory is not writeable. https://err.sh/zeit/next.js/build-dir-not-writeable

### DISCUSSION THREAD

**Discussion by nimitwalia89** [vercel/next.js#13091] [Help]

**Comment by theCuriousOne** [comment]
@nimitwalia89 What Node.js version do you have? I experience the same error with Node.js 8 but not with Node.js 10 (or greater)

**Comment by nimitwalia89** [comment]
@theCuriousOne updating works. thanks :)

**Comment by vaibhav1802** [comment]
I was facing similar error (directory doesn't exist error errno: -2, code: 'ENOENT', syscall: 'mkdir') in next build because of old nodejs version (8). Updated to 12 and issue got fixed.

**Comment by Rashmi-278** [comment]
I am facing still this issue,
Node version : v12.18.3
OS : Linux
kernel : 4.19.0-9-cloud-amd64
PRETTY_NAME="Debian GNU/Linux 10 (buster)"
NAME="Debian GNU/Linux"
VERSION_ID="10"
VERSION="10 (buster)"
VERSION_CODENAME=buster

Im using Amazon lightsail instance

**Comment by theCuriousOne** [comment]
@Rashmi-278 is the error the same as above? what commands are you executing?
